test/expected_header

```diff
@@ -0,0 +1,6 @@
+HTTP/1.1 000
```

Comment:
Did you intend to update https://github.com/google/nginx-sxg-module/blob/master/.github/workflows/end-to-end-test.yml to compare against this?

Reply:
Sorry, `test/expected_header` is forgotten file to add on the previous patch (I wonder why the test passed).

Reply:
I think the test passed because https://github.com/google/nginx-sxg-module/blob/3e11910d5ab3b51f0c05e446240104ad1cd8eee2/.github/workflows/end-to-end-test.yml#L50 does not depend on `expected_header`. Is there another test that does?